ui/src/components/layouts/MapLayout.tsx

```diff
@@ -76,7 +76,11 @@ function MapLayout({ children }: { children: ReactNode }) {
         flexDirection: "column",
       }}
     >
-      <Header />
+      <Header
+        onBrandClick={() => {
+          setDrawerOpen(true);
```

Comment:
Should this just open the drawer, or should it also navigate to the index page? I'm not clear from the original issue (#326) what "home screen" is meant to imply

Reply:
This is just adding on to the original action. It's a `Link` to `/` already. This just also opens the drawer.

Reply:
Good point, I didn't catch that